### PR TITLE
Fix/styles in popup

### PR DESCRIPTION
**Pull request by Yojuks**

### CODE REVIEW

File: frontend/src/components/PointPopup/IconsComponent/IconsComponent.tsx [hunk 1]
```diff
@@ -58,18 +58,21 @@ export const IconsComponent: FC<Props> = ({
     <>
       <IconButton onClick={e => handleRating(e, 'likes')}>
         {rating.likes.includes(userId) ? (
-          <ThumbUpIcon fontSize="small" />
+          <ThumbUpIcon fontSize="small" sx={{ marginRight: '5px' }} />
         ) : (
-          <ThumbUpOutlinedIcon fontSize="small" />
+          <ThumbUpOutlinedIcon fontSize="small" sx={{ marginRight: '5px' }} />
```
Comment (steadyGuy) [inline frontend/src/components/PointPopup/IconsComponent/IconsComponent.tsx]: mr: 1. I think it's much better.

Comment (steadyGuy) [inline frontend/src/components/PointPopup/IconsComponent/IconsComponent.tsx]: mr: 1

File: frontend/src/components/PointPopup/IconsComponent/IconsComponent.tsx [hunk 2]
```diff
@@ -58,18 +58,21 @@ export const IconsComponent: FC<Props> = ({
...
         )}
         {rating.likes.length}
       </IconButton>
 
       <IconButton onClick={e => handleRating(e, 'dislikes')}>
         {rating.dislikes.includes(userId) ? (
-          <ThumbDownIcon fontSize="small" />
+          <ThumbDownIcon fontSize="small" sx={{ marginRight: '5px' }} />
```
Comment (steadyGuy) [inline frontend/src/components/PointPopup/IconsComponent/IconsComponent.tsx]: mr: 1

Comment (steadyGuy) [inline frontend/src/components/PointPopup/IconsComponent/IconsComponent.tsx]: mr: 1

File: frontend/src/components/PointPopup/IconsComponent/IconsComponent.tsx [hunk 3]
```diff
@@ -58,18 +58,21 @@ export const IconsComponent: FC<Props> = ({
...
         ) : (
-          <ThumbDownAltOutlinedIcon fontSize="small" />
+          <ThumbDownAltOutlinedIcon
+            fontSize="small"
+            sx={{ marginRight: '5px' }}
```
Comment (steadyGuy) [inline frontend/src/components/PointPopup/IconsComponent/IconsComponent.tsx]: mr: 1

**Review by steadyGuy — APPROVED** [review]

File: frontend/src/components/PointPopup/IconsComponent/IconsComponent.tsx (hunk 1 above)
Comment (Yojuks) [reply frontend/src/components/PointPopup/IconsComponent/IconsComponent.tsx]: done

File: frontend/src/components/PointPopup/IconsComponent/IconsComponent.tsx (hunk 2 above)
Comment (Yojuks) [reply frontend/src/components/PointPopup/IconsComponent/IconsComponent.tsx]: done

Comment (Yojuks) [reply frontend/src/components/PointPopup/IconsComponent/IconsComponent.tsx]: done

File: frontend/src/components/PointPopup/IconsComponent/IconsComponent.tsx (hunk 3 above)
Comment (Yojuks) [reply frontend/src/components/PointPopup/IconsComponent/IconsComponent.tsx]: done

**Review by naberalgamma — APPROVED** [review]

**Review by vasylneshchadimov — COMMENTED** [review]
bag


The pop-up window does not open below before it opens. I think it's because the image in file has the right pading

**Review by vasylneshchadimov — COMMENTED** [review]
Delete location don't work

**Comment by Yojuks** [comment]
> Delete location don't work

Because this  branch created before that Nikita merged him task

**Pull request merged**